packages/python-argus/PKGBUILD

```diff
@@ -0,0 +1,48 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=python-argus
+_pkgname=Argus
+__pkgname=argus
+pkgver=7.56ece87
+pkgrel=1
+pkgdesc='All-in-One OSINT Toolkit.'
```

Comment:
Could you set a more precise description rather than "it's an OSINT tool" please?

Reply:
i'm not sure how else to describe it. that's literally from it's readme. it has too many features to list. it's similar to csi linux's osint toolset.